lib/storage/index.js

```diff
@@ -10,7 +10,7 @@ const {
   ClientDetails,
   NoteOptions,
 } = require('../../types')
-const fetch = require('isomorphic-fetch')
+const fetch = require('cross-fetch')
```

Comment:
If you have to make this update in 5 places today might be worth consolidating this behind a Tozny wrapper for fetch

Reply:
Going to leave as is for the time being. Two reasons:

1. This is a full dependency change because isomorphic-fetch stopped receiving updates. This is very rare in general, but these statements are more like importing golang packages into several files.
2. When we switch to typescript we'll switch to ES6 imports, so much of this import code will get a refactor at that point.